deb/debian/dirs

```diff
@@ -1 +1,2 @@
+/etc/stackdriver/
```

Comment:
According to https://www.debian.org/doc/manuals/maint-guide/dother.en.html#dirs, do we need this file at all?

Reply:
You're right. We don't. PTAL